components/BottomNav.js

```diff
@@ -83,7 +34,7 @@ export default function BottomNav() {
         inactiveColor="#cecece"
         shifting="true"
         labeled="true"
-        barStyle={{ backgroundColor: "#030D12" }}
+        barStyle={{ backgroundColor: "#030D12", height: 56 }}
```

Comment:
is it possible to use variables for css colors? if yes i would suggest to use them :)